shell/src/main/java/alluxio/cli/fsadmin/command/report/SummaryCommand.java

```diff
@@ -32,24 +36,16 @@
  * Print Alluxio cluster summarized information.
  * This class depends on meta master client and block master client.
  */
-public class SummaryCommand {
-  /** Print Alluxio cluster summarized information. */
-  public static void printSummary() {
+public class SummaryCommand extends AbstractCommand {
+
+  @Override
+  public int run(CommandLine cl) throws IOException {
     System.out.println("Alluxio Cluster Summary: ");
 
     int numOfSpaces = 4;
     String spaces = String.format("%" + numOfSpaces + "s", "");
 
-    printMetaMasterInfo(spaces);
-    printBlockMasterInfo(spaces);
-  }
-
-  /**
-   * Prints the meta master info.
-   *
-   * @param spaces for better print out message indentations
-   */
-  private static void printMetaMasterInfo(String spaces) {
+    // Prints Alluxio meta master information
```

Comment:
we use active voice (Prints) for javadoc, but not usually for code comments (Print)

Reply:
It is strange, in many files in our code base. We use Constructs, Returns, Retrieves for code comments. 
It should use Construct, Return, Retrieve instead?

Reply:
So you mean like that Checks in doc and Check in comment
```java
  /**
   * Checks if the summary command output is valid.
   *
   * @param output the summary command output
   */
  private void checkSummaryResults(String output) {
    // Check if meta master values are available
    String expectedMasterAddress = NetworkAddressUtils
        .getConnectAddress(ServiceType.MASTER_RPC).toString();
```
Understand, I will change it, thanks